.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: python -m pip install --upgrade -r requirements-test.txt && go get && go build ./... && go test ./...
```

Comment:
Looks like this file was added by accident

Reply:
Not accidentaly, but I do think it's good for open source projects to support Gitpod setups to make contributions quicker and easier. I can remove it if it's not something you are interested in for this project though.

Reply:
I like anything that makes contributions easier :). Maybe move this to a separate pr so that we can discuss and make sure maintainers are aware.